qkernel/src/syscalls/sys_proxy.rs

```diff
@@ -819,7 +819,7 @@ fn CudaMemcpyAsync(
     kind: CudaMemcpyKind,
     stream: u64,
 ) -> Result<i64> {
-    // error!("CudaMemcpyAsync: count:{}, kind{}", count, kind);
+    // error!("CudaMemcpyAsync: count:{:x}, kind{}, src:{:x}, dst:{:x}", count, kind, src, dst);
```

Comment:
If it is not needed, please consider removing commented out code (same for below).